test/examples/type_library.raml

```diff
@@ -0,0 +1,10 @@
+#%RAML 1.0 Library
+
+types:
+  CatWithAddress:
+    additionalProperties: false
+    properties:
+      address: string
+something:
```

Comment:
```
[type_library.raml:7:0] ERROR Unknown node: 'something'
```
this is not valid RAML. the `something` node would have to be under `types:`

Reply:
This has been corrected.